src/content/en/2020/privacy.md

```diff
@@ -47,29 +47,39 @@ We use [WhoTracksMe](https://whotracks.me/)'s tracker list to determine the perc
 We examined the most widely used trackers and plot the prevalence of the 10 most popular ones.
 
 {{ figure_markup(
-  image="privacy-biggest-third-party-trackers.png",
-  caption="Top 10 Trackers",
-  description="Bar chart showing the prevalence of the 10 most popular trackers used on mobile and desktop clients. There is little difference between desktop and mobile and mobile has 65.9% for google_analytics, 65.5% for googleapis.com, 63.3% for gstatic, 58.3% for google_fonts, 50.0% for doubleclick, 47.6% for google, 42.4% for google_tag_manage, 30.9% for facebook, 19.2% for google_adservices, and 13.1% for cloudflare.",
+  image="privacy-biggest-third-party-potential-trackers.png",
+  caption="Top 10 Potential Trackers",
+  description="Bar chart showing the prevalence of the 10 most popular potential trackers used on mobile and desktop clients. There is little difference between desktop and mobile and mobile has 65.5% for google_analytics, 65.9% for googleapis.com, 63.3% for gstatic, 58.3% for google_fonts, 50.0% for doubleclick, 47.6% for google, 42.4% for google_tag_manager, 30.9% for facebook, 19.2% for google_adservices, and 12.7% for cloudflare.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=850649042&format=interactive",
-  sheets_gid="1677398038",
-  sql_file="top100_cookies_set_from_header.sql"
+  sheets_gid="1677398038"
   )
 }}
 
-The largest player on the online tracking market is without doubt Google, with eight of its tracking domains present in the top 10 trackers and prevalent on at least 70% of websites. They are followed are Facebook and Cloudflare–though the latter is probably more reflective of the popularity of them as a hosting site.
+The largest player on the online tracking market is without doubt Google, with eight of its domains present in the top 10 potential trackers and prevalent on at least 70% of websites. They are followed by Facebook and Cloudflare–though the latter is probably more reflective of the popularity of them as a hosting site.
+
+WhoTracksMe's tracker list also defines categories that the trackers belong to. If we remove CDNs and Hosting sites from our statistics, under the assumption they may not track—or at least that that is not their primary function—then you get a slightly different view of the top 10.
```

Comment:
"under the assumption that they may not track" sounds a bit weaker than what I'd consider ideal. We are talking about CDN domains that are often cookieless. Might be interesting to scan HA to see if those CDN domains have cookies set on them, and if they don't, clarify that they are not tracking today, but are "potential trackers" as they have the power to start tracking in the future (which IIRC is the reasoning).

Reply:
Understand your concerns @yoavweiss but I would still prefer to err on the side of caution here given the chapters topic and the power these entities could wield in this space. I think your suggested proposal to scan HA would be limited in nature, and many do set cookies for LoadBalancing or WAF reason. Plus cookies are far from the only way of tracking (particularly for hosting providers with access to IP addresses and the like).

My initial thought was to include a link to Google Fonts FAQ about this as an example with an explicit comment like "and [some of these providers have statements they do not track"](https://developers.google.com/fonts/faq2#what_does_using_the_google_fonts_api_mean_for_the_privacy_of_my_users)" but on re-reading that, I'm not sure that's what it really says so I find that a little weaker, so thought more confusing to include, hence went with above. If that FAQ or privacy policy was stronger in this regards, I think we could be stronger too.

I've tried to be present an independent and balanced view here, and certainly think it's an improvement on just including them as trackers without comment - but it's gonna be difficult to make everyone happy!

@ydimova @KenjiBaheux what's your view here? Guessing you'll both be on either side of this argument! 🙂

Reply:
> I think your suggested proposal to scan HA would be limited in nature, and many do set cookies for LoadBalancing or WAF reason.

Sure. But if they don't set cookies, that's a strong indication.

> Plus cookies are far from the only way of tracking (particularly for hosting providers with access to IP addresses and the like)

That's fair. That would've been a different story if e.g. the relevant snippets included a `[referrerPolicy=no-referrer](https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-no-referrer)` attribute, but that's not typically the case.



> I've tried to be present an independent and balanced view here, and certainly think it's an improvement on just including them as trackers without comment

Agree that it's a significant improvement. Just think that it can be improved further... :)